week-2/mandatory/2-ecommerce-db/task.md

```diff
@@ -8,7 +8,20 @@ Below you will find a set of tasks for you to complete to set up a database for
 
 To submit this homework write the correct commands for each question here:
 ```sql
-
+1. select * from customers where country = 'United States';
+2. select * from customers order by name;
+3. select * from products where product_name like '%socks%';
+4. select a.prod_id, p.product_name,a.supp_id,a.unit_price from product_availability a join products p on p.id = a.prod_id where a.unit_price > 100;
+5. select * from product_availability order by unit_price DESC limit 5;
```

Comment:
You have correctly identified the 5 most expensive products. Could you do a join here to also include the product name?

Reply:
'''sql
select a.supp_id, a.unit_price, p.product_name from product_availability a join products p on p.id = a.prod_id order by a.unit_price DESC limit 5;
'''
is this what you mean?